### DISCUSSION TITLE
[Feature request] Allow user to specify save data directory

### DISCUSSION THREAD

**Discussion by ipkpjersi** [rafaelvcaetano/melonDS-android#858] [Ideas]
Hi,

Now that the desktop version of MelonDS supports users specifying save data directory, I think it would be amazing if the Android version of MelonDS supported this. This could be accomplished by manually adding this feature, or by updating MelonDS Android to 0.9.4 of MelonDS.

Cheers.

**Comment by rafaelvcaetano** [comment]
Hi!

This feature has been available in the Android port since almost the beginning. You can choose separate folders for save files and save states in Settings > Save Files. Yo need to disable the "Save next to ROM file" and select the directory using "Save file directory".

**Reply by ipkpjersi** [reply]
Oh, I don't know how I missed that feature, thanks! :smile:
